<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crossTerm</class>
 <widget class="QWidget" name="crossTerm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="verticalFrame">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>140</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="CONNECT_BUTTON">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RESCAN_BUTTON">
          <property name="text">
           <string>ReScan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="HELP_BUTTON">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ABOUT_BUTTON">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="QUIT_BUTTON">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="COM_PORT">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>COM Port</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="COM_PORT_SELECTOR"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="BAUD_RATE">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>140</height>
        </size>
       </property>
       <property name="title">
        <string>Baud Rate</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="1">
         <widget class="QRadioButton" name="BAUD_19200">
          <property name="text">
           <string>19200</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="BAUD_1200">
          <property name="text">
           <string>1200</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="BAUD_28800">
          <property name="text">
           <string>28800</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="BAUD_38400">
          <property name="text">
           <string>38400</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="BAUD_2400">
          <property name="text">
           <string>2400</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QRadioButton" name="BAUD_56000">
          <property name="text">
           <string>56000</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="BAUD_9600">
          <property name="text">
           <string>9600</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="BAUD_600">
          <property name="text">
           <string>600</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="BAUD_14400">
          <property name="text">
           <string>14400</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="BAUD_4800">
          <property name="text">
           <string>4800</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="BAUD_57600">
          <property name="text">
           <string>57600</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="BAUD_115200">
          <property name="text">
           <string>115200</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QRadioButton" name="BAUD_128000">
          <property name="text">
           <string>128000</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QRadioButton" name="BAUD_256000">
          <property name="text">
           <string>256000</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QRadioButton" name="BAUD_CUSTOM">
          <property name="text">
           <string>custom</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="DATA_BITS">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>140</height>
        </size>
       </property>
       <property name="title">
        <string>Data Bits</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QRadioButton" name="DATA_BITS_6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="DATA_BITS_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="DATA_BITS_8">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="DATA_BITS_7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="PARITY">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>140</height>
        </size>
       </property>
       <property name="title">
        <string>Parity</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="2" column="0">
         <widget class="QRadioButton" name="PARITY_EVEN">
          <property name="text">
           <string>even</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="PARITY_NONE">
          <property name="text">
           <string>none</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="PARITY_ODD">
          <property name="text">
           <string>odd</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="PARITY_MARK">
          <property name="text">
           <string>mark</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="PARITY_SPACE">
          <property name="text">
           <string>space</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="STOP_BITS">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>140</height>
        </size>
       </property>
       <property name="title">
        <string>Stop Bits</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="STOP_BITS_1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="STOP_BITS_1_5">
          <property name="text">
           <string>1.5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="STOP_BITS_2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="HANDSHAKING">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>140</height>
        </size>
       </property>
       <property name="title">
        <string>Handshaking</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,1,1,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="HANDSHAKING_NONE">
          <property name="text">
           <string>none</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="HANDSHAKING_RTS_CTS">
          <property name="text">
           <string>RTS/CTS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="HANDSHAKING_XON_XOFF">
          <property name="text">
           <string>XON/XOFF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="HANDSHAKING_RTS_CTS_XON_XOFF">
          <property name="text">
           <string>RTS/CTS+XON/XOFF</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="HANDSHAKING_RTS_ON_TX">
            <property name="text">
             <string>RTS on TX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="HANDSHAKING_INVERT">
            <property name="text">
             <string>invert</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>CheckBox</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPlainTextEdit" name="RECEIVE_EDIT">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string notr="true"/>
         </property>
        </widget>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>18</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPlainTextEdit" name="TRANSMIT_EDIT">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="sendFileFrame" native="true">
     <layout class="QHBoxLayout" name="sendFileLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="FILE_EDIT">
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CHOOSE_FILE">
        <property name="text">
         <string>Choose File</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="sendFileSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="byteDelayLabel">
        <property name="text">
         <string>Byte Delay(ms):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="BYTE_DELAY_SPIN_BOX"/>
      </item>
      <item>
       <widget class="QCheckBox" name="LINE_BY_LINE_CHECK_BOX">
        <property name="text">
         <string>Line by Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FILE_CONTROL_BUTTON">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
